### DISCUSSION TITLE
[Projects] Add number of Todo/In Progress tasks after the project name

### DISCUSSION THREAD

**Discussion by RupertBarrow** [community/community#15929] [Projects and Issues]
Wherever the project name appears (as a title, or in a list), add the number of tasks which are in Status Todo/In Progress after the project name

**Comment by Neudrino** [comment]
This issue is probably a **duplicate** of the most popular issue:
- :arrow_right: https://github.com/github-community/community/discussions/7575

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050
- https://github.com/github-community/community/discussions/15929

An **answer** from Github as of 2022-06-13 can be found at: https://github.com/github-community/community/discussions/16721#discussioncomment-2939031
